nmdc_automation/api/nmdcapi.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, site_configuration: Union[str, Path, SiteConfig]):
     def refresh_token(func):
         def _get_token(self, *args, **kwargs):
             # If it expires in 60 seconds, refresh
-            if not self.token or self.expires_at + 60 > time():
+            if not self.token or self.expires_at < time() - 60:
```

Comment:
I read this new version as: "If ... or the token expires before 60 seconds ago." With that in mind, I am wondering whether you meant to write:

```suggestion
            if not self.token or self.expires_at < time() + 60:
```

I read that as "If ... or the token expires within the next 60 seconds."